doc/api/timers.md

```diff
@@ -525,6 +525,9 @@ added:
 * `delay` {number} The number of milliseconds to wait before resolving the
   promise.
 * `options` {Object}
+  * `ref` {boolean} Set to `false` to indicate that the scheduled `Timeout`
+    should not require the Node.js event loop to remain active.
+    **Default:** `true`.
```

Comment:
Worth noting that this option is not in the draft spec – but actually, `.wait` is not even in it, the closest thing is `postTask` which has a quite different shape: https://wicg.github.io/scheduling-apis/#schedule-a-posttask-task